### DISCUSSION TITLE
Plans for supporting lower RAM dependency

### DISCUSSION THREAD

**Discussion by cayolblake** [Chia-Network/bladebit#38] [Q&A]
Hello,

Is it possible to have Bladebit support RAM lower than 416 GB of RAM ? 

If so, what would be the impact?

**Comment by mocksoul** [comment]
It could be done, but will require a lot of extra data move/copy in RAM, this will lead to much more ctx switches inside CPU cores and will be slower. You already have madmax for that, it requires ~270G ram and is 10-20% slower than bladebit.

**Reply by cayolblake** [reply]
Makes sense.
